doc/source/tokudb/tokudb_installation.rst

```diff
@@ -57,9 +53,9 @@ You can disable the transparent huge pages by running the following command as r
 Installation
 ============
 
-TokuDB storage engine for |Percona Server| is currently available in our :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+|TokuDB| storage engine for |Percona Server| is currently available in our :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
```

Comment:
just for the reference, this is still accurate as you need to install separate tokudb package